app/server/lib/OIDCConfig.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Configuration for OpenID Connect (OIDC), useful for enterprise single-sign-on logins.
+ * A good informative overview of SAML is at https://developer.okta.com/blog/2019/10/21/illustrated-guide-to-oauth-and-oidc
```

Comment:
Nit: SAML -> OpenID Connect

```suggestion
 * A good informative overview of OIDC is at https://developer.okta.com/blog/2019/10/21/illustrated-guide-to-oauth-and-oidc
```